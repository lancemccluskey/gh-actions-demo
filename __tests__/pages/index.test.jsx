import { render, screen } from '@testing-library/react';
import Homepage from '../../pages/index';

describe('<Homepage />', () => {
  it('renders', () => {
    const { container } = render(<Homepage />);

    expect(container).toMatchSnapshot();
  });

  it('should have GH Actions title', () => {
    render(<Homepage />);

    expect(screen.getByText('GH Actions')).toBeInTheDocument();
  });
});
