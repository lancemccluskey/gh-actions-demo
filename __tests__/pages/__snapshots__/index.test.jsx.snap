// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Homepage /> renders 1`] = `
<div>
  <div
    class="css-14vyic8-Homepage"
  >
    <div
      class="css-17svcfk-Homepage"
    >
      <h1
        class="css-us1uot-Homepage"
      >
        GH Actions
      </h1>
    </div>
  </div>
</div>
`;
